Library/Homebrew/cmd/update.rb

```diff
@@ -398,10 +398,26 @@ def select_formula(key)
 
   def dump_formula_report(key, title)
     formula = select_formula(key)
+    formula_installed = select_installed_formula(key, formula)
     formula.map! { |oldname, newname| "#{oldname} -> #{newname}" } if key == :R
     unless formula.empty?
       ohai title
-      puts_columns formula
+      puts_columns(formula, formula_installed)
```

Comment:
You may want to update `puts_columns` as well. Currently, it only appends a `*` for highlight entires. You may want to check it using [`Tty`](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/utils.rb#L11-L71)